### ISSUE TITLE
Keyboard focus on system tray

### ISSUE THREAD

**Issue by brettinternet** [i3/i3#4202]
## I'm submitting a…

<pre>
[ ] Bug
[x] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior

Keyboard focus cannot be given to the system tray or i3status elements on the i3bar

This appears to be incorrect:

```sh
i3-msg '[class="i3bar"] focus'  # doesn't work
```

## Desired Behavior

I would like to move keyboard focus to i3bar or the system tray and tab through items for accessibility. I'm primarily interested in accessing system tray content without a mouse.

## Environment

```
i3 version: 4.18.2 (2020-07-26)
i3bar version:  4.18.2 (2020-07-26)
i3status: 2.13
```

```
- distro: Arch
- compositor: yshui/picom
```

In most desktop environments, such as GNOME, KDE and Ubuntu, it appears to be possible to focus the status bar. However, perhaps there are limitations with a WM? From my unsuccessful search on this topic, I am assuming this feature has not been implemented.

---

Thank you for providing such wonderful software. I benefit tremendously from this project.

**Comment by orestisfl** [comment]
What are you trying to achieve exactly?

**Comment by brettinternet** [comment]
Thank you for the reply. My objective is to focus and navigate or tab through the system tray with the keyboard when a mouse is not accessible. This could look similar to the way one might tab through a website's header links.

**Comment by Airblader** [comment]
I don't think this is something that's going to happen because allowing i3bar (or any dock client) to receive focus would cause a lot of usability issues. Directional focus commands, moving the mouse across monitors or momentarily over the bar etc.

I'm also not sure how that would actually work with Xembed since we embed the client but i3bar isn't actually in control of those windows.

**Comment by stapelberg** [comment]
Yep, seems really tricky on a technical level, unfortunately.

I’ll close this, but feel free to re-open if you think we missed any evidence to the contrary.
